src/future/backports/email/feedparser.py

```diff
@@ -34,7 +34,7 @@
 
 NLCRE = re.compile('\r\n|\r|\n')
 NLCRE_bol = re.compile('(\r\n|\r|\n)')
-NLCRE_eol = re.compile('(\r\n|\r|\n)\Z')
+NLCRE_eol = re.compile(r'(\r\n|\r|\n)\Z')
 NLCRE_crack = re.compile('(\r\n|\r|\n)')
```

Comment:
These regexes look pretty broken to be honest. All of these should be raw string literals. There are probably more examples of this in the codebase.